### PR TITLE
Prepare prod Overcloud for Octavia deployment

### PR DESCRIPTION
**Pull request by paulbrowne**
Create SG, flavor, network and COE cluster changes for Octavia in production cluster

### CODE REVIEW

File: etc/cumulus-config/cumulus-config.yml [hunk 1]
```diff
@@ -659,6 +690,13 @@ cumulus_flavors:
   - "{{ cumulus_flavor_lsst_medium }}"
   - "{{ cumulus_flavor_bm_dirac25y }}"
 
+cumulus_flavor_octavia_small:
+  name: "octavia.small"
+  ram: 1024
+  disk: 20
```
Comment (JohnGarbutt) [inline etc/cumulus-config/cumulus-config.yml]: Sadly this means we will run out of local disk, tiny was the smallest unit I could manage without wasting RAM with the size of our disk :(

File: etc/cumulus-config/cumulus-config.yml [hunk 2]
```diff
@@ -892,29 +931,46 @@ cumulus_container_clusters_template_swarm_fedora_atomic_27:
...
+  network_driver: "flannel"
+  docker_storage_driver: "overlay2"
+  server_type: "vm"
+
+cumulus_container_clusters_template_k8s_147:
   public:
   floating-ip-disabled:
-  labels: "cgroup_driver=cgroupfs,ingress_controller=traefik,prometheus_monitoring=false,heat_container_agent_tag=train-dev,kube_tag=v1.14.6,cloud_provider_tag=v1.14.0"
-  external_network: "{{ cumulus_network_internal_name }}"
+  labels: "cgroup_driver=cgroupfs,ingress_controller=octavia,monitoring_enabled=true,tiller_enabled=true,heat_container_agent_tag=train-stable-1,kube_tag=v1.14.7,cloud_provider_tag=v1.14.0,min_node_count=1,max_node_count=5,master_lb_floating_ip_enabled=true,auto_scaling_enabled=true,tiller_tag=v2.14.3,autoscaler_tag=v1.0"
```
Comment (JohnGarbutt) [inline etc/cumulus-config/cumulus-config.yml]: ah, this is where it is hiding... I need this I think.

File: etc/cumulus-config/cumulus-config.yml [hunk 3]
```diff
@@ -929,16 +929,16 @@ cumulus_container_clusters_template_swarm_fedora_atomic_27:
...
 
-cumulus_container_clusters_template_k8s_148:
+cumulus_container_clusters_template_k8s_146:
   public:
-  floating-ip-disabled:
-  labels: "cgroup_driver=cgroupfs,ingress_controller=octavia,monitoring_enabled=true,tiller_enabled=true,heat_container_agent_tag=train-stable-1,kube_tag=v1.14.8,cloud_provider_tag=v1.14.0,min_node_count=1,max_node_count=5,master_lb_floating_ip_enabled=true,auto_scaling_enabled=true,tiller_tag=v2.14.3,autoscaler_tag=v1.0"
-  master_lb_enabled:
+  #floating-ip-disabled: need octavia to avoid this
+  labels: "cgroup_driver=cgroupfs,ingress_controller=traefik,monitoring_enabled=true,tiller_enabled=true,heat_container_agent_tag=train-stable-1,kube_tag=v1.14.6,cloud_provider_tag=v1.14.0,min_node_count=1,max_node_count=5,master_lb_floating_ip_enabled=true,auto_scaling_enabled=true,tiller_tag=v2.14.3,autoscaler_tag=v1.0"
+  #master_lb_enabled: need octavia?
```
Comment (paulbrowne) [inline etc/cumulus-config/cumulus-config.yml:954]: Ah, so this was put in pre-supposing Octavia, but we can always change it back when Octavia is deployed.

File: etc/cumulus-config/cumulus-config.yml (hunk 1 above)
Comment (paulbrowne) [reply etc/cumulus-config/cumulus-config.yml]: Well, the DIB image spat out for the amphorae has virtual size 5GB, so we should be able to take the disk on the flavor down to that to save on local disk.

**Review by JohnGarbutt — APPROVED** [review]

**Pull request merged**